deploy/alphabet.go

```diff
@@ -55,22 +55,12 @@ func designateNeoFSAlphabet(ctx context.Context, prm initAlphabetPrm) error {
 
 		prm.logger.Info("checking NeoFS Alphabet role of the committee members...")
 
-		accsWithAlphabetRole, err := roleContract.GetDesignatedByRole(noderoles.NeoFSAlphabet, prm.monitor.currentHeight())
+		ok, err := checkRole(noderoles.NeoFSAlphabet, prm.blockchain, prm.monitor, prm.committee)
 		if err != nil {
-			prm.logger.Error("failed to check role of the committee, will try again later", zap.Error(err))
-			continue
+			return fmt.Errorf("role check: %w", err)
```

Comment:
do not change behavior: log and `continue`, not return. Same in `enableNotary`

Reply:
i just deleted this part and wrote from scratch so this is why i missed it. overall, it is strange to me: only caller's context can stop this then, all the errors are just ignored (how do we expect to check rolles, if no network is available?). just a user's experience issue to me. however, i am for keeping behavior in commits that do not declare changes, fixed